recipes/pbsim/meta.yaml

```diff
@@ -0,0 +1,16 @@
+package:
+  name: pbsim
+  version: "1.0.3"
+
+about:
+  home: https://code.google.com/archive/p/pbsim/
+  license: GPLv2
+  summary: "PBSIM simulates PacBio reads"
+
+source:
+  url: https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/pbsim/pbsim-1.0.3.tar.gz
+  md5: c070394d42b6e4cdae0bb2aaf4856782
+
```

Comment:
It is in the guidelines. We had gcc removed some time ago, but now we share the build environment with conda forge, and they haven't yet made this step. If you find the time, please add gcc and also libgcc.